test/case/infix_containers/container_phys/topology.dot

```diff
@@ -1 +0,0 @@
-../container_bridge/topology.dot
```

Comment:
Why this?

Reply:
I did not actively go in and remove the newline, if that's what you're asking?  I've renamed/simplified the test directory names and both these files are symlinks to the container bridge test's `topology.dot` file.

Reply:
Aha, you thought I'd removed them?  Just GIT having a bad day I think, didn't manage to track the rename:

```
 test/case/infix_containers/container_phys/topology.dot                               |  1 -
 test/case/infix_containers/container_veth/topology.dot                               |  1 -
 test/case/infix_containers/phys/topology.dot                                         |  1 +
 test/case/infix_containers/veth/topology.dot                                         |  1 +
```